drivers/counter/counter_qmsi_aonpt.c

```diff
@@ -247,7 +247,7 @@ static int aon_timer_init(struct device *dev)
 
 static const struct aonpt_config aonpt_conf_info = {
 	.info = {
-		.max_wrap = UINT32_MAX,
+		.max_top_value = UINT32_MAX,
 		.freq = 32768,
 		.count_up = false,
 		.channels = 1,
```

Comment:
i missed that in the previous review: i see that set_alarm return `-ENODEV`. That indicates that there is 0 channels. you should change to `.channels=0` and completely remove `aon_timer_qmsi_set_alarm` implementation since `counter_set_channel_alarm` is checking number of channels before calling implementation.